scipy/stats/stats.py

```diff
@@ -4638,33 +4644,55 @@ def kruskal(*args):
 
     References
     ----------
-    .. [1] http://en.wikipedia.org/wiki/Kruskal-Wallis_one-way_analysis_of_variance
+    .. [1] Kruskal, William H. & Wallis, W. Allen, "Use of Ranks in
```

Comment:
And ``Volume`` as ``Vol.``? Both minor, but good to keep things consistent with the numpy docstring guidelines.